setup/docker.md

```diff
@@ -158,6 +158,40 @@ multi-machine scaling.
 
 For organizations, we recommend one Docker host per team of 5-10 developers.
 
+## Docker Compose
+
+   For a convenent way to run Coder in Docker, you can use [Docker Compose](https://docs.docker.com/compose/).
+
+   Simply create a new directory (for example, named `c4d`) and place the following
+   content in a file named `docker-compose.yml`:
+
+   ```yaml
+
+version: "3.5"
+services:
+  coder:
+    image: docker.io/codercom/coder:1.28.2
+    container_name: coderd
+    restart: unless-stopped
+    ports:
+      - 7080:7080/tcp
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - ${HOME}/.coder:/var/run/coder
+
+   ```
+
+   Then, navigate to that folder in your terminal and run the command:
+
+   ```shell-session
+
+   docker-compose up -d
```

Comment:
Should we be recommending `docker compose up -d`?

Reply:
I intentionally chose the `docker-compose` syntax in case folks on older versions want to try this. The new syntax is backwards-compatible:

```
$ ls -l $(which docker-compose)
lrwxr-xr-x 1 root wheel 68 Feb 27 21:05 /usr/local/bin/docker-compose -> /Applications/Docker-4.5.0.app/Contents/Resources/bin/docker-compose

```